forge/db/views/Application.js

```diff
@@ -1,14 +1,18 @@
 module.exports = {
     application: function (app, application) {
-        const raw = application.toJSON()
-        const filtered = {
-            id: raw.hashid,
-            name: raw.name,
-            createdAt: raw.createdAt,
-            updatedAt: raw.updatedAt,
-            links: raw.links
+        if (application) {
```

Comment:
Should this responsible for guarding against application being `null`? That feels like it belongs in whatever view is calling this view.

Reply:
This was self-consistent with the Team view that does the same thing. I'm okay with it as-is.